plugin/src/test/java/io/jenkins/plugins/casc/misc/Util.java

```diff
@@ -203,4 +209,40 @@ public static String convertToJson(String yamlString) {
         JSONObject jsonObject=new JSONObject(map);
         return jsonObject.toString();
     }
+
+    /**
+     * Returns the JSON V7 schema for the running jenkins instance.
+     * Example Usage:
+     *      * <pre>{@code
+     *      * Schema jsonSchema = returnSchema();}
+     *      * </pre>
+     * @return
+     */
+    public static Schema returnSchema() {
+        JSONObject schemaObject = generateSchema();
+        JSONObject jsonSchema = new JSONObject(
+            new JSONTokener(schemaObject.toString()));
+        return SchemaLoader.load(jsonSchema);
+    }
+
+    /**
+     *Validates a given jsonObject against the schema generated for the current live jenkins instance
+     *      * Example Usage:
+     *      *      * <pre>{@code
+     *      *      * assertTrue(validateSchema(jsonSubject));}
+     *      *      * </pre>
+     * @param jsonSubject
+     * @return boolean
+     */
+    public static boolean validateSchema(JSONObject jsonSubject) {
+        final Logger LOGGER = Logger.getLogger(TokenReloadAction.class.getName());
+        final String failureMessage  = "The YAML file provided for this schema is invalid";
+        try {
+            returnSchema().validate(jsonSubject);
+        } catch (Exception ie) {
+            LOGGER.warning(failureMessage + ie);
```

Comment:
always log the stacktrace

```suggestion
            LOGGER.warning("The YAML file provided for this schema is invalid", ie);
```

Reply:
This takes only one parameter, I am not sure how to put two of them together

Reply:
something like:
```
LOGGER.level(Level.WARNING, message, e)
```